internal/alert/provider/webhook/send.go

```diff
@@ -0,0 +1,74 @@
+package webhook
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/balerter/balerter/internal/alert/message"
+	"github.com/balerter/balerter/internal/config"
+)
+
+func (w *Webhook) Send(message *message.Message) error {
+	ctx, cancel := context.WithTimeout(context.Background(), w.conf.Timeout)
+	defer cancel()
+
+	req, err := w.request(ctx, message)
+	if err != nil {
+		return err
+	}
+
+	client := &http.Client{
+		Timeout: w.conf.Timeout,
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
```

Comment:
I think, we should to allow users check response code (with config). For example, if webhook returns 4xx - return error for send Alert